### PR TITLE
Fix showing PROTOBUF schema in versions tab

### LINKED ISSUE #1188: schema versions tab crashes when schema type is PROTOBUF
The UI will crash when the user navigates to the versions tab of a schema that has a type of PROTOBUF.

The issue is the JSON.parse function on the PROTOBUF schema.

### PR DESCRIPTION
**Pull request by ali-1995**
Fixes #1188

### CODE REVIEW

File: client/src/containers/Schema/SchemaDetail/SchemaVersions/SchemaVersions.jsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ class SchemaVersions extends Root {
         return {
           id: schema.id,
           version: schema.version,
-          schema: JSON.stringify(JSON.parse(schema.schema), null, 2)
+          schema: "PROTOBUF" === formData.schemaType?latestSchemaVersion.schema :JSON.stringify(JSON.parse(latestSchemaVersion.schema), null, 2);
```
Comment (tchiotludo) [inline client/src/containers/Schema/SchemaDetail/SchemaVersions/SchemaVersions.jsx:38]: This one will not work I think, there is no formdata on this page

**Review by tchiotludo — CHANGES_REQUESTED** [review]

File: client/src/containers/Schema/SchemaDetail/SchemaVersions/SchemaVersions.jsx (hunk 1 above)
Comment (ali-1995) [reply client/src/containers/Schema/SchemaDetail/SchemaVersions/SchemaVersions.jsx]: OOPS! I forgot to rename variable names. I will fix this and commit the changes after testing. Sorry!

Comment (ali-1995) [reply client/src/containers/Schema/SchemaDetail/SchemaVersions/SchemaVersions.jsx]: I've tested the code and fix the variable names.

**Comment by tchiotludo** [comment]
thanks :+1:

**Pull request merged**